contracts/rewards/src/error.rs

```diff
@@ -1,9 +1,12 @@
 use axelar_wasm_std::IntoContractError;
-use cosmwasm_std::OverflowError;
+use cosmwasm_std::{OverflowError, StdError};
 use thiserror::Error;
 
 #[derive(Error, Debug, PartialEq, IntoContractError)]
 pub enum ContractError {
+    #[error(transparent)]
+    Std(#[from] StdError),
```

Comment:
where is this needed? As far as I can tell it's wrapped everywhere already

Reply:
Map::update requires this for some reason: https://docs.rs/cw-storage-plus/latest/cw_storage_plus/struct.Map.html#method.update